scripts/presubmit-tests.sh

```diff
@@ -304,6 +304,15 @@ function main() {
       echo ">> docker version"
       docker version
     fi
+    # TODO(chaodaiG): node/pod names are missing after migrating from bootstrap to podutil,
```

Comment:
Better explain the reason, then add a TODO pointing to an issue that, once resolved, allow us to remove this.

Reply:
done, it's a `TODO` in k8s/test-infra code instead of issue